agent/config/types.go

```diff
@@ -20,6 +20,10 @@ import (
 	cnitypes "github.com/containernetworking/cni/pkg/types"
 )
 
+// ImagePullBehaviorType is an enum variable type corresponding to different agent pull
+// behaviors including default, always, never and once.
+type ImagePullBehaviorType int8
```

Comment:
why an int8 over an int?

Reply:
Because it's just an enum variable, which doesn't require a large range, there is usage of `int8` elsewhere in agent:
https://github.com/aws/amazon-ecs-agent/blob/159ae5c3067e0a0d7387ff68af2a28bc90f3baaa/agent/acs/update_handler/updater.go#L65